lib/translate/common.h

```diff
@@ -12,12 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "absl/strings/string_view.h"
+#include "lib/scsi_defs.h"
+
 namespace translator {
 
+constexpr absl::string_view kNvmeVendorIdentification = "NVMe    ";
+static void (*debug_callback)(const char *);
+
 enum class StatusCode { kSuccess, kInvalidInput, kNoTranslation, kFailure };
 
-void DebugLog(const char* format, ...);
+void DebugLog(const char *format, ...);
```

Comment:
We should all agree on a standard for this style 
https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

Reply:
I just did clang-format, we should stick with what that automatically does.

Reply:
weird, i createad a new `test1(const char *format, ...)` function and clang-format corrected it to `test1(const char* format, ...)` . It didn't reformat `test2(const char* format, ...)`

Reply:
I'm having the same results @eyhhuang

Reply:
oh maybe it was a merge. we should stick to char* format then.

Reply:
Final decision was to use format `type* var_name`